File: src/main/java/hilfling/backend/hilfling/service/GangService.java
package hilfling.backend.hilfling.service;

import hilfling.backend.hilfling.model.Gang;
import hilfling.backend.hilfling.repositories.GangRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

@Service
public class GangService extends
        GenericBaseServiceImplementation<Gang> {
    @Autowired
    GangRepository repository;
    public GangRepository getRepository() {
        return repository;
    }

};

File: src/main/java/hilfling/backend/hilfing/service/GangService.kt
package hilfling.backend.hilfing.service

import hilfling.backend.hilfing.model.Gang
import hilfling.backend.hilfing.repositories.GangRepository
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Service

@Service
class GangService  {
    @Autowired
    var repository: GangRepository? = null
    public override fun getRepository(): GangRepository {
        return repository!!
    }
}

// GetAll()
// GetById()
// Create()
// Delete
// Update